### PR TITLE
Optimize call to FeatureEnabled API

### PR DESCRIPTION
**Pull request by adilhafeez**

### CODE REVIEW

**Comment by rodaine** [comment]
@adilhafeez is this covered by the existing tests? 
@junr03 Do we not have Travis set up on this project?

File: snapshot/snapshot.go [hunk 1]
```diff
@@ -44,7 +44,13 @@ func New() (s *Snapshot) {
 }
 
 func (s *Snapshot) FeatureEnabled(key string, defaultValue uint64) bool {
-	return defaultRandomGenerator.Random()%100 < min(s.GetInteger(key, defaultValue), 100)
+	featureValue := s.GetInteger(key, defaultValue)
+	if featureValue == 100 {
```
Comment (chriselion) [inline snapshot/snapshot.go]: nit: check `>= 100` here, and remove the `min()` below.

**Comment by rodaine** [comment]
Travis has been enabled now, so fixing @chriselion's comments should trigger a build.

**Comment by junr03** [comment]
@rodaine thanks for enabling, I was trying to get access to enable it.

**Comment by junr03** [comment]
@adilhafeez closing. Please address comments and re open if you want to merge

**Pull request closed without merging**